tst/EnergyPlus/unit/SizingManager.unit.cc

```diff
@@ -558,3 +559,3733 @@ TEST_F(EnergyPlusFixture, SizingManager_OverrideAvgWindowInSizing)
+        "    ,                        !- Watts per Person {W/person}",
+        "    0,                       !- Fraction Latent",
+        "    0.3000000,               !- Fraction Radiant",
+        "    0;                       !- Fraction Lost",
+
+        "  Schedule:Compact,",
+        "    Evening,                 !- Name",
+        "    ,                        !- Schedule Type Limits Name",
+        "    Through: 12/31,          !- Field 9",
+        "    For: Alldays,            !- Field 10",
+        "    Until: 16:00,0.0,         !- Field 11",
+        "    Until: 20:00,1.0,        !- Field 11",
+        "    Until: 24:00,0.0;        !- Field 11",
+
+        "ZoneAirHeatBalanceAlgorithm,",
+        "ThirdOrderBackwardDifference,  !- Algorithm",
+        "No,                      !- Do Space Heat Balance for Sizing",
+        "No;                       !- Do Space Heat Balance for Simulation",
+        "  Output:Table:SummaryReports,",
+        "      AllSummary;                             !- Report Name 1",
+    });
+
+    ASSERT_TRUE(process_idf(idf_objects));
+    SimulationManager::ManageSimulation(*state);
+
+    // For coincident, expect zone Des Cooling Load to be less than sum of space loads which is 832.44 + 940.2 + 810.65 = 2583.29
+    EXPECT_EQ("31553.14", OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnHtCalcDesLd, "ZONE 1"));
+    EXPECT_EQ("0.876", OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnHtCalcDesAirFlow, "ZONE 1"));
+    EXPECT_EQ("0.876", OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnHtUserDesAirFlow, "ZONE 1"));
+    EXPECT_EQ("CHICAGO_IL_USA ANNUAL HEATING 99% DESIGN CONDITIONS DB",
+              OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnHtDesDay, "ZONE 1"));
+    EXPECT_EQ("1/21 08:00:00", OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnHtPkTime, "ZONE 1"));
+    EXPECT_EQ("19423.64", OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnClCalcDesLd, "ZONE 1"));
+    EXPECT_EQ("1.348", OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnClCalcDesAirFlow, "ZONE 1"));
+    EXPECT_EQ("1.348", OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnClUserDesAirFlow, "ZONE 1"));
+    EXPECT_EQ("CHICAGO_IL_USA ANNUAL COOLING 1% DESIGN CONDITIONS DB/MCWB",
+              OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnClDesDay, "ZONE 1"));
+    EXPECT_EQ("7/21 16:00:00", OutputReportPredefined::RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchZnClPkTime, "ZONE 1"));
+}
+TEST_F(EnergyPlusFixture, SizingManager_ZoneSizing_Coincident_NonAir_10x_Latent_SpaceHB)
```

Comment:
And finally, add a big latent load to one space to get a different cooling sizing result.